homeassistant/components/devolo_home_network/__init__.py

```diff
@@ -128,3 +175,18 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+def require_reauth(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """Require the user to log in again."""
+    hass.add_job(
+        hass.config_entries.flow.async_init(
```

Comment:
Use this helper instead to deduplicate flows:

https://github.com/home-assistant/core/blob/a332cd8abad5eba07f52f122cca1e26bda3f91ff/homeassistant/config_entries.py#L690-L697